src/EnergyPlus/HVACManager.cc

```diff
@@ -2237,20 +2237,32 @@ void ReportInfiltrations(EnergyPlusData &state)
             thisInfiltration.InfilTotalLoss = -TotalLoad;
         }
         // CR7751  second, calculate using indoor conditions for density property
-        AirDensity = Psychrometrics::PsyRhoAirFnPbTdbW(state, state.dataEnvrn->OutBaroPress, thisSpaceHB.MAT, thisSpaceHB.airHumRatAvg, RoutineName);
-        thisInfiltration.InfilVdotCurDensity = thisInfiltration.InfilMdot / AirDensity;
+        Real64 const spAirDensity = Psychrometrics::PsyRhoAirFnPbTdbW(state, state.dataEnvrn->OutBaroPress, thisSpaceHB.MAT, thisSpaceHB.airHumRatAvg, RoutineName);
+        thisInfiltration.InfilVdotCurDensity = thisInfiltration.InfilMdot / spAirDensity;
         thisInfiltration.InfilVolumeCurDensity = thisInfiltration.InfilVdotCurDensity * TimeStepSysSec;
-        Real64 vol = state.dataHeatBal->space(spaceNum).Volume;
-        if (vol > 0.0) {
-            thisInfiltration.InfilAirChangeRate = thisInfiltration.InfilVolumeCurDensity / (TimeStepSys * vol);
-        } else {
-            thisInfiltration.InfilAirChangeRate = 0.0;
-        }
 
         // CR7751 third, calculate using standard dry air at nominal elevation
-        AirDensity = state.dataEnvrn->StdRhoAir;
-        thisInfiltration.InfilVdotStdDensity = thisInfiltration.InfilMdot / AirDensity;
+        Real64 const stdAirDensity = state.dataEnvrn->StdRhoAir;
+        thisInfiltration.InfilVdotStdDensity = thisInfiltration.InfilMdot / stdAirDensity;
```

Comment:
Could probably just eliminate this local? I'm not sure if this is more efficient or not. At least it's const. Several places.

Reply:
Done